src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -244,3 +244,65 @@ def test_configure_waveform_acquisition(session):
     assert session.range == 100
     assert session.waveform_rate == 100000
     assert session.waveform_points == 400
+
+
+def test_fetch_waveform(session):
+    session.configure_waveform_acquisition(nidmm.Function.WAVEFORM_VOLTAGE, 10, 1000, 10)
+    with session.initiate():
+        number_of_points_to_read = 10
+        measurements = session.fetch_waveform(array_size=number_of_points_to_read)[1]
+        assert measurements == number_of_points_to_read
```

Comment:
Shouldnt this be 'len(measurements) == number_of_points_to_read' ?

Reply:
measurement is actualNumberOfPoints parameter returned by fetch_multiple. So we don't need len(measurements) here. 

To make test more clear will update test with both variables.